packages/components/psammead-caption/src/index.jsx

```diff
@@ -17,9 +17,9 @@ import { scriptPropType } from '@bbc/gel-foundations/prop-types';
 import { getSansRegularItalic } from '@bbc/psammead-styles/font-styles';
 
 const Caption = styled.figcaption`
-  ${props => (props.script ? getLongPrimer(props.script) : '')};
-  color: ${C_METAL};
+  ${({ script }) => script && getLongPrimer(script)};
   ${({ service }) => getSansRegularItalic(service)}
```

Comment:
Are we confident that these don't crash in the same way as the get type size functions? should these be `service && getX(service)`?